src/manifold/src/impl.cpp

```diff
@@ -488,8 +488,7 @@ void Manifold::Impl::Hull(const std::vector<glm::vec3>& vertPos) {
   Vec<glm::dvec3> pointCloudVec(numVert);
   manifold::transform(vertPos.begin(), vertPos.end(), pointCloudVec.begin(),
                       [](const glm::vec3& v) { return glm::dvec3(v); });
-  QuickHull qh(pointCloudVec);
-  ConvexHull hull = qh.getConvexHullAsMesh(pointCloudVec, false);
+  ConvexHull hull = getConvexHullAsMesh(pointCloudVec, false);
```

Comment:
You might just merge `getConvexHullAsMesh` into `Impl::Hull` and put this combined implementation in `quickhull.cpp`. Then you can remove `ConvexHull` and directly construct `halfedge_` and `vertPos_`.

Reply:
I have merged `getConvexHullAsMesh` into `Impl::Hull` and removed `ConvexHull`. I think it's better to keep the `Hull` function inside `Impl`, since it follows a similar flow to the other functions inside `Impl`. Although if you think it's better to put it into `quickhull.cpp`, I can make that change.

Reply:
I mean both - notice how our `Impl` functions are implemented across many files? You can implement `Impl::Hull` in `quickhull.cpp` if you want, so long as it imports `impl.h`.